go.mod

```diff
@@ -31,23 +30,34 @@ require (
 	k8s.io/client-go v12.0.0+incompatible
 	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
 	kubevirt.io/client-go v0.34.2
-	kubevirt.io/containerized-data-importer v1.23.5
+	kubevirt.io/containerized-data-importer v1.39.0
 	sigs.k8s.io/controller-runtime v0.9.7
 	sigs.k8s.io/yaml v1.2.0
 )
 
 replace (
+	// required by Microsoft/hcsshim, containers/storage, sriov-network-operator
+	// Not used within this Operator.
+	// Bump to avoid CVE detection with v1.5.4. https://bugzilla.redhat.com/show_bug.cgi?id=1899487, https://bugzilla.redhat.com/show_bug.cgi?id=1982681
```

Comment:
"with 1.5.4" - is prolly a typo meant to be some older version?

Reply:
@bogdando good eye. just pushed a follow up patch to update/fix this comment. Thanks